CHANGELOG.md

```diff
@@ -8,4 +8,13 @@
 
 ## 0.0.3
 
-- Fix broken gifs in ReadMe [#3](https://github.com/woltapp/wolt_modal_sheet/pull/3)
\ No newline at end of file
+- Fix broken gifs in ReadMe [#3](https://github.com/woltapp/wolt_modal_sheet/pull/3)
+
+## 0.0.4
+
+- Fix repository link in pubspec file [#5](https://github.com/woltapp/wolt_modal_sheet/pull/5)
+- Prevent horizontal scroll in the main content causing top bar motion [#13](https://github.com/woltapp/wolt_modal_sheet/pull/13)
+- Make modal type builder optional by providing default breakpoint and modal type builder [#14](https://github.com/woltapp/wolt_modal_sheet/pull/14)
+- Add callback for on modal dismiss with drag to be utilized in Navigator 2.0 [#15](https://github.com/woltapp/wolt_modal_sheet/pull/15)
+- Rename page list builder notifier parameter for WoltModalSheet.show method [#16](https://github.com/woltapp/wolt_modal_sheet/pull/16)
+- Scale hero image from center in HeroImageFlowDelegate [#17](https://github.com/woltapp/wolt_modal_sheet/pull/16)
```

Comment:
The link should be `pull/17` :)

Reply:
Thanks! Done. https://github.com/woltapp/wolt_modal_sheet/pull/19/commits/991650ae31166bc11cebed15f08dcc778534bae7